custom/enikshay/nikshay_datamigration/factory.py

```diff
@@ -17,9 +17,9 @@
 TEST_CASE_TYPE = 'test'
 
 
-def validate_number(string_value):
+def validate_phone_number(string_value):
     if string_value is None or string_value.strip() == '':
-        return None
+        return ''
     else:
         return int(string_value)
```

Comment:
good point about `int` @nickpell I think we can just keep as a string? 

This is actually for an import script, and the data is pretty messy. 

@nickpell do you think it would be worthwhile to do something like [this](https://github.com/dimagi/commcare-hq/blob/mk-enikshay-nikshay-api/custom/enikshay/integrations/ninetyninedots/repeater_generators.py#L133-L138)? to format the phone number properly or do you think that would lead to too many failures?

Right now, the numbers we collect explicitly do not have a country code attached, but I'll raise this with the requirements team and see if that is something we need. I didn't realize that was a requirement to use the messaging framework.